contrib/gitian-debian/build.sh

```diff
@@ -22,11 +22,30 @@ tarball=bitcoin-$ver-linux64.tar.gz
 tar xzvf ../$tarball
 mv bitcoin-$ver usr
 
+# copy bitcoinxt.service file to lib/systemd/system directory
+mkdir -p lib/systemd/system 
+cp ../bitcoinxt.service lib/systemd/system
+
+# copy bitcoin.conf file to etc/bitcoinxt 
+mkdir -p etc/bitcoinxt
+cp ../bitcoin.conf etc/bitcoinxt
+
+# create file to force creation of data folder
+mkdir -p var/lib/bitcoinxt
+touch var/lib/bitcoinxt/.empty
+
 # Rename the binaries so we don't conflict with regular Bitcoin
 mv usr/bin/bitcoind usr/bin/bitcoinxt
 mv usr/bin/bitcoin-cli usr/bin/bitcoinxt-cli
 mv usr/bin/bitcoin-tx usr/bin/bitcoinxt-tx
 
+# Remove unneeded files 
+rm usr/bin/bitcoin-qt
```

Comment:
Should we maybe just rename bitcoin-qt instead of removing it? Some users seem to really want it.